grpc-google-cloud-build-v1/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+  <difference>
+    <!-- TODO: remove after 1.0.4 released -->
+    <differenceType>6001</differenceType>
+    <className>com/google/cloudbuild/v1/*Grpc</className>
+    <field>METHOD_*</field>
+  </difference>
+  <difference>
+    <differenceType>7012</differenceType>
+    <className>com/google/cloudbuild/v1/*OrBuilder</className>
+    <method>* get*(*)</method>
+  </difference>
+  <difference>
+    <differenceType>7012</differenceType>
+    <className>com/google/cloudbuild/v1/*OrBuilder</className>
+    <method>boolean contains*(*)</method>
+  </difference>
+  <difference>
+    <differenceType>7012</differenceType>
+    <className>com/google/cloudbuild/v1/*OrBuilder</className>
+    <method>boolean has*(*)</method>
+  </difference>
```

Comment:
These are needed in the proto artifact rather than here.

Reply:
oh, sorry, didn't notice it